src/main/java/org/vietabroader/view/verifier/SheetVerifier.java

```diff
@@ -0,0 +1,25 @@
+package org.vietabroader.view.verifier;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class SheetVerifier extends InputVerifier {
```

Comment:
This is a good idea. Use another name since `SheetVerifier` is kinda misleading. Maybe `TextFieldVerifier`, or just `TextVerifier` ?